pom.xml

```diff
@@ -25,7 +25,7 @@
   </licenses>
 
   <properties>
-    <aws-java-sdk.version>1.11.844</aws-java-sdk.version>
+    <aws-java-sdk.version>1.12.3</aws-java-sdk.version>
```

Comment:
Can you make this diff a separate PR? This change is independent of the release, so we should include the AWS SDK upgrade PR in the release notes for any customers who might need to override their SDK. [Example previous PR](https://github.com/awslabs/amazon-kinesis-client/commit/1d5099cef4459f2f1a4804d5daa2ccf0352f3a6d), [example previous release notes](https://github.com/awslabs/amazon-kinesis-client/commit/9db3a11ffe04bdac412732afa408bccc8629183b).

Reply:
Updated this PR to contain only SDK version bump.

Reply:
Thanks!